Cannot find module 'debug'

Hi I'm looking forward to use this addon but get the following at the install command. 

```
Cannot find module 'debug'
Error: Cannot find module 'debug'
    at Function.Module._resolveFilename (module.js:336:15)
    at Function.Module._load (module.js:286:25)
    at Module.require (module.js:365:17)
    at require (module.js:384:17)
    at Object.<anonymous> (/Users/cugerone/Workspace/flightright/website/claimcheck/node_modules/ember-cli-polyfill-io/index.js:3:15)
    at Module._compile (module.js:434:26)
    at Object.Module._extensions..js (module.js:452:10)
    at Module.load (module.js:355:32)
    at Function.Module._load (module.js:310:12)
    at Module.require (module.js:365:17)
    at require (module.js:384:17)
```

What package am I missing? I use ember cli 2.8 with node v4

Thank you in advance for your feedback

--- a/package.json
+++ b/package.json
@@ -54,5 +54,8 @@
   "files": [
     "index.js",
     "lib/index.js"
-  ]
+  ],
+  "dependencies": {
+    "debug": "^2.6.8"
+  }
 }

